pkg/operator/flags.go

```diff
@@ -74,5 +75,6 @@ func DefaultOperatorFlags() map[string]string {
 		GuardrailsDeployManaged:            FlagFalse,
 		CloudProviderConfigEnabled:         FlagTrue,
 		EtcHostsEnabled:                    FlagFalse,
+		EtcHostsManaged:                    FlagFalse,
```

Comment:
This should be set to True, as this default only applies to new clusters (existing clusters will "default" to False for all newly added flags by virtue of returning nil).

```suggestion
		EtcHostsManaged:                    FlagTrue,
```